README.md

```diff
@@ -24,9 +24,17 @@ git clone https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-probl
 
 ```bash
 cd nurse-scheduling-problem-solver
+```
+3. Install dependecies
+```
+$ julia 
+julia> ]
+(@v1.5) pkg> activate .
+(@v1.5) pkg> instantiate
```

Comment:
tutaj nie będzie @v1.5, tylko nazwa projektu xD

Reply:
U mnie to wygląda dokładnie tak jak zapisałem wyżej. Proponujesz żebym zamienił na `nurse-scheduling-solver>`, podobnie do tego jak jest nazwane repo, czy nazwa projektu się rózni ?

Reply:
to coś robisz zle
![image](https://user-images.githubusercontent.com/26227269/97920487-d18e7080-1d59-11eb-923e-a6bc779b43ca.png)

Reply:
Robiłem, dobrze, ale niedopatrzyłem. 
Rzeczywiście była nazwa projektu